dsl_parser/yaml_loader.py

```diff
@@ -67,9 +67,9 @@ def construct_yaml_set(self, node):
         return self._holder(obj, node)
 
     def construct_yaml_str(self, node):
-        obj = SafeConstructor.construct_yaml_str(self, node)
+        obj = self.construct_scalar(node)
         try:
-            obj = str(obj)
+            obj.encode('ascii')
```

Comment:
we still keep the assumption that blueprints have to be ascii
maybe we'll drop it soon

Reply:
Is there any reason to keep this assumption, or just that we need to test before changing this?

Reply:
From discussion: We'll leave this as-is now, because a change like this (while it shouldn't break anything) does need testing to make sure it actually accomplishes something useful rather than erroring out in an apparently unrelated place much deeper in the codebase.

Reply:
no real reason but let's keep behaviour changes outside the scope of this